backpack-common/src/main/java/net/skyscanner/backpack/calendar2/CalendarParams.kt

```diff
@@ -79,12 +79,24 @@ data class CalendarParams(
         /**
          * Only a single, non-disabled date can be selected.
          */
-        data object Single : SelectionMode
+        data class Single(
```

Comment:
these are shared between view & compose. i think we need to add a comment saying that the accessibility labels are NOT supported in the view version

Reply:
Done!